### PR TITLE
Changed to anker tags & added hyauctions for items

### PR DESCRIPTION
**Pull request by Ekwav**

### CODE REVIEW

File: src/components/OptionsMenu/OptionsMenu.tsx [hunk 1]
```diff
@@ -18,12 +18,18 @@ function OptionsMenu(props: Props) {
     let { trackEvent } = useMatomo();
     let [anchorEl, setAnchorEl] = useState(null);
 
+    const getItemTag = (isItemPage: number) => {
```
Comment (matthias-luger) [inline src/components/OptionsMenu/OptionsMenu.tsx:21]: use the useParams hook.
let { tag} = useParams();

File: src/components/OptionsMenu/OptionsMenu.tsx [hunk 2]
```diff
@@ -18,12 +18,18 @@ function OptionsMenu(props: Props) {
...
+        return window.location.href.substr(isItemPage + 6).split('/')[0].split('?')[0];
+    }
+    
     let available: AvailableLinks[] = [];
-    const isItemPage = window.location.href.indexOf("/item/") > 0;
```
Comment (matthias-luger) [inline src/components/OptionsMenu/OptionsMenu.tsx:22]: isItemPage seems like a boolean. Let it be a boolean

File: src/components/OptionsMenu/OptionsMenu.tsx [hunk 3]
```diff
@@ -50,7 +56,7 @@ function OptionsMenu(props: Props) {
         <span>
             <div className="d-none d-md-block">
                 {available.map((result, i) => (
-                    <Button variant="outlined" onClick={(e: any) => { navigate(result.url) }}>{result.title}</Button>
+                    <a href={result.url} title={result.title} target="_blank"><Button variant="outlined" >{result.title}</Button></a>
```
Comment (matthias-luger) [inline src/components/OptionsMenu/OptionsMenu.tsx:59]: use the <Link to=""/>-element

**Review by matthias-luger — CHANGES_REQUESTED** [review]
fix comments

File: src/components/OptionsMenu/OptionsMenu.tsx (hunk 2 above)
Comment (Ekwav) [reply src/components/OptionsMenu/OptionsMenu.tsx:22]: reverted

**Pull request merged**